core/src/instance.rs

```diff
@@ -570,7 +634,7 @@ pub mod tests {
     pub fn can_process_action() {
         let mut instance = Instance::new(test_context("jason"));
 
-        let context = instance.initialize_context(test_context("jane"));
```

Comment:
Pretty sure that removing this call (and the ones below) to `initialize_context` is what makes the tests fail..